File: easy-solution/minesp3164/python/977.py
import math

class Solution(object):
    def sortedSquares(self, nums):
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        for i in range(len(nums)):
            nums[i] = int(math.pow(nums[i], 2))
        nums.sort()
        return nums

File: easy-solution/bear-bear-bear/977.rs
// TODO: O(n) 으로 처리해보기
impl Solution {
    pub fn sorted_squares(mut nums: Vec<i32>) -> Vec<i32> {
        nums.sort_unstable_by_key(|x| x.abs());
        nums.iter_mut().for_each(|x| *x = x.pow(2));
        nums
    }
}
